README.md

```diff
@@ -1,3 +1,202 @@
 # FTPropertyWrappers
 
+![Swift](https://github.com/futuredapp/FTPropertyWrappers/workflows/Swift/badge.svg)
 
+Commonly used features implemented in Swift's property wrapper layer. This package contains wrappers for User Defaults, Keychain, Publisher and synchronization.
```

Comment:
Commonly used by whom? I would make it clear why we made it and it's us who are using it.

Publisher is Combine provided type. I named it StoredSubject for a reason.